programs/light/tests/tests.rs

```diff
@@ -1,179 +1,97 @@
-        program_id: account_compression::ID,
-        accounts: vec![
-            AccountMeta::new(signer_pubkey, true),
-            AccountMeta::new(group_accounts, false),
-            AccountMeta::new_readonly(system_program::ID, false),
-        ],
-        data: instruction_data.data(),
-    };
-    instruction
-}
-
-pub fn create_update_authority_instruction(
-    signer_pubkey: Pubkey,
-    authority_pda: (Pubkey, u8),
-    group_account: Pubkey,
-    new_authority: Pubkey,
-) -> Instruction {
-    let update_authority_ix = light::instruction::UpdateAuthority {
-        bump: authority_pda.1,
-        new_authority,
-    };
-
-    // update with new authority
-    let instruction = Instruction {
-        program_id: light::ID,
-        accounts: vec![
-            AccountMeta::new(signer_pubkey, true),
-            AccountMeta::new(authority_pda.0, false),
-            AccountMeta::new(group_account, false),
-            AccountMeta::new_readonly(account_compression::ID, false),
-        ],
-        data: update_authority_ix.data(),
-    };
-    instruction
-}
-
-pub fn create_register_program_instruction(
-    signer_pubkey: Pubkey,
-    authority_pda: (Pubkey, u8),
-    group_account: Pubkey,
-    program_id_to_be_registered: Pubkey,
-) -> (Instruction, Pubkey) {
-    let registered_program_pda =
-        Pubkey::find_program_address(&[program_id_to_be_registered.to_bytes().as_slice()], &ID).0;
-
-    let register_program_ix = light::instruction::RegisterSystemProgram {
-        bump: authority_pda.1,
-        program_id: program_id_to_be_registered,
-    };
-
-    let instruction = Instruction {
-        program_id: light::ID,
-        accounts: vec![
-            AccountMeta::new(signer_pubkey, true),
-            AccountMeta::new(authority_pda.0, false),
-            AccountMeta::new(group_account, false),
-            AccountMeta::new_readonly(account_compression::ID, false),
-            AccountMeta::new_readonly(system_program::ID, false),
-            AccountMeta::new(registered_program_pda, false),
-        ],
-        data: register_program_ix.data(),
-    };
-    (instruction, registered_program_pda)
-}
-
-#[tokio::test]
-async fn test_create_and_update_group() {
-    let mut program_test = ProgramTest::default();
-    println!("account compression ID: {:?}", ID);
-    program_test.add_program("light", light::ID, None);
-    println!(" light::ID: {:?}", light::ID);
-
-    program_test.add_program("account_compression", ID, None);
-    program_test.set_compute_max_units(1_400_000u64);
-    let authority_pda = Pubkey::find_program_address(
-        &[light::AUTHORITY_PDA_SEED, &light::ID.to_bytes().as_slice()],
-        &light::ID,
-    );
-    let mut context = program_test.start_with_context().await;
-    let updated_keypair = Keypair::new();
-
-    airdrop_lamports(&mut context, &updated_keypair.pubkey(), 1_000_000_000)
+pub const LIGHT_ID: Pubkey = pubkey!("5WzvRtu7LABotw1SUEpguJiKU27LRGsiCnF5FH6VV7yP");
```

Comment:
You can probably import it from test-utils, same with the other `pubkey!` constants

Reply:
actually I can just take it from it's own crate light::ID;